docs/Summary.md

```diff
@@ -83,6 +83,7 @@ Acuminator does not perform static analysis of projects whose names contain `Tes
 | [PX1074](diagnostics/PX1074.md) | `PXSetupNotEnteredException` cannot be thrown in any event handlers except for the `RowSelected` event handlers. | Warning (ISV Level 1: Significant) | Unavailable |
 | [PX1075](diagnostics/PX1075.md) | `PXCache.RaiseExceptionHandling` cannot be invoked from the `FieldDefaulting`, `FieldSelecting`, `RowSelecting`, and `RowPersisted` event handlers. | Error | Unavailable |
 | [PX1076](diagnostics/PX1076.md) | This code calls Acumatica internal API marked with PXInternalUseOnlyAttribute which is not intended for public use | Warning | Unavailable |
+| [PX1077](diagnostics/PX1077.md) | Event handlers in graphs and graph extensions should be `protected virtual` methods | Error | Available |
```

Comment:
@EAndrosova please review the summary for PX1077 diagnostic

Reply:
Please change to the following:
Event handlers in graphs and graph extensions should have the `protected` and `virtual` modifiers.

Reply:
Done.